### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by DuncanLithgow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# BIM7AA_Properties_DataPack
+
+This property set includes Revit shared Parameters for BIM7AA parameters.
+
+The BIM7AA excel spreadsheet found at [bim7aa.dk/BIM7AA_Typekodning](http://www.bim7aa.dk/BIM7AA_Typekodning.html) includes mapping tables from BIM7AA to several other systems:
+
+   * IFC2x3 TC1 CV2.0 ( [Industry Foundation Classes IFC2x Edition 3 Technical Corrigendum 1](https://standards.buildingsmart.org/IFC/RELEASE/IFC2x3/TC1/HTML/) ) 
+   * CCS ( [Cuneco Classification System](https://molio.dk/produkter/digitale-vaerktojer/gratis-vaerktojer/ccs-cuneco-classification-system)). CCS wil be replaced in the future by CCI 
+   * FVK ( [Forvaltningklassifaktion](https://www.lbf.dk/om-lbf/lbf-orienterer/2009/8/forvaltnings-klassifikation/)
```
Comment (tsolsen) [inline README.md]: reference er [Forvaltningsklassifikation](https://www.lbf.dk/publikationer/2014-forvaltnings-klassifikation/)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+   * BC SfB ( [Samarbetskommitten for Byggnadsfragor)](https://www.designingbuildings.co.uk/wiki/CI/SfB)
```
Comment (tsolsen) [inline README.md]: reference er [BC/SfB-bygningsdelstavle](https://molio.dk/boeger/sfb-1988-bygningsdeltavle/c-23/p-2193)

**Review by tsolsen — CHANGES_REQUESTED** [review]
Se kommentarer.

**Comment by DuncanLithgow** [comment]
Sorry for pushing so many separate commits, I thought I was only saving to my master. I have a lot to learn about git and github.

**Pull request merged**